From 04165366515a2ba36c78540da776d3a12164f824 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Sat, 20 Mar 2021 16:19:01 +0900
Subject: [PATCH] cifsd: add index.rst in cifs documentation
Git-commit: 04165366515a2ba36c78540da776d3a12164f824
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Since more than one file is in the cifs document directory,
This patch add an index.rst.

Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 Documentation/filesystems/cifs/index.rst | 10 ++++++++++
 Documentation/filesystems/index.rst      |  3 +--
 2 files changed, 11 insertions(+), 2 deletions(-)
 create mode 100644 Documentation/filesystems/cifs/index.rst

diff --git a/Documentation/filesystems/cifs/index.rst b/Documentation/filesystems/cifs/index.rst
new file mode 100644
index 000000000000..e762586b5dc7
--- /dev/null
+++ b/Documentation/filesystems/cifs/index.rst
@@ -0,0 +1,10 @@
+===============================
+CIFS
+===============================
+
+
+.. toctree::
+   :maxdepth: 1
+
+   cifsd
+   cifsroot
diff --git a/Documentation/filesystems/index.rst b/Documentation/filesystems/index.rst
index ac9396f2bb8a..bdba80ae2bb1 100644
--- a/Documentation/filesystems/index.rst
+++ b/Documentation/filesystems/index.rst
@@ -72,8 +72,7 @@ Documentation for filesystem implementations.
    befs
    bfs
    btrfs
-   cifs/cifsd
-   cifs/cifsroot
+   cifs/index
    ceph
    coda
    configfs
-- 
2.33.0

